src/lib/feathersClient.js

```diff
@@ -18,9 +19,16 @@ export const socket = io(config.feathersConnection, {
 });
 
 // socket IO error events
-socket.on('connect_error', _e => console.log('Could not connect to FeatherJS'));
-socket.on('connect_timeout', _e => console.log('Could not connect to FeatherJS: Timeout'));
-socket.on('reconnect_attempt', _e => console.log('Trying to reconnect to FeatherJS: Timeout'));
+socket.on('connect_error', err => ErrorPopup('Could not connect to FeatherJS', err));
+socket.on('connect_timeout', () =>
+  ErrorPopup('Could not connect to FeatherJS: Timeout. Please try and refresh the page.'),
+);
+socket.on('reconnect_attempt', () =>
+  ErrorPopup('Trying to reconnect to FeatherJS: Timeout. Please try and refresh the page.'),
+);
+socket.on('error', err =>
+  ErrorPopup('Trying to reconnect to FeatherJS: Timeout. Please try and refresh the page.', err),
+);
```

Comment:
this might be a bit annoying for the user, if connection drops and reconnects are attempted.

In my opinion we should do this silently and only show an error if x time has passed or x number of reconnects attempted

Reply:
Should I just make it silently for now?